Update translations for Vietnamese

vhpx opened the pull request:
## What kind of change does this PR introduce?

README I18n update for Vietnamese

## What is the current behavior?

Missing Vietnamese translations for README file in i18n/

## What is the new behavior?

Added new Vietnamese translations for README file in i18n/

## Additional context

none

vhpx committed:
Update translations for Vietnamse

vhpx committed:
Merge branch 'supabase:master' into master

owlnai commented on i18n/README.vi-vn.md:
The rest of languages should be removed, read the HTML comment above: 
> *Remove this list if you're traslating to another language, it's hard to keep updated across multiple files*

phucvo0708 replied:
Resolved. Thanks for the heads-up!

vhpx committed:
Updated translations list

Removed redundant list of translations as it is harder to keep track across multiple files.

vhpx committed:
Fixed typos & updated missing link @ master/README

vhpx reviewed (commented):
Changes reviewed. LGTM.

owlnai reviewed (approved):
LGTM.

phamhieu commented:
@phucvo0708 thank you for the PR. 

However, this looks like a google translate version 🥺

I add comments on the status/client info. It will be great if you can help to check and rephrase programming terms in Vnmese if not then using the english terms will be better.

phucvo0708 commented:
Thanks for the feedback on my PR.
@phamhieu 

You stated that:
> I add comments on the status/client info. It will be great if you can help to check and rephrase programming terms in Vnmese if not then using the english terms will be better.
I can't seem to find any comments you mentioned.

Could you add a ref link to that so I could check it out and do some before-hand programming term conversions to improve the readability and clarification on my proposed Vietnamese translations?

phamhieu commented:
Sorry, i used github mobile and didn't realise that the review is pending

vhpx committed:
Update README.vi-vn.md

Fix awkward Vietnamese translations

phucvo0708 commented on i18n/README.vi-vn.md:
```diff
@@ -0,0 +1,88 @@
+
+#### Thư viện khách hàng
+
+Các thư viện khách hàng của chúng tôi là các mô-đun. Mỗi thư viện con là một triển khai độc lập cho một hệ thống bên ngoài. Đây là một trong những cách chúng tôi hỗ trợ các công cụ hiện có.
```
Resolved. Please check out the new version of it and see if it's in better context now.

phucvo0708 commented on i18n/README.vi-vn.md:
```diff
@@ -0,0 +1,88 @@
+- [x] Beta công khai: Đủ ổn định cho hầu hết các trường hợp sử dụng không phải trong môi trường doanh nghiệp
+- [ ] Công khai: Sẵn sàng cho môi trường production doanh nghiệp
+
+Chúng tôi hiện đang ở giai đoạn Beta công khai. Xem "releases" của repo này để nhận thông báo về các bản cập nhật lớn.
```
Resolved. Please check out the new version of it and see if it's in better context now.

phamhieu commented on i18n/README.vi-vn.md:
```diff
@@ -0,0 +1,88 @@
+
+## Cách Supabase hoạt động
+
+Supabase là sự kết hợp của các công cụ mã nguồn mở. Chúng tôi đang xây dựng các tính năng của Firebase bằng cách sử dụng các sản phẩm mã nguồn mở, cấp doanh nghiệp. Nếu các công cụ và cộng đồng tồn tại, với MIT, Apache 2 hoặc giấy phép mở tương đương, chúng tôi sẽ sử dụng và hỗ trợ công cụ đó. Nếu công cụ không tồn tại, chúng tôi tự xây dựng và mở mã nguồn của nó. Supabase không phải là phiên bản ...
+
+**Kiến trúc hiện tại**
+
+Supabase là một [nền tảng được lưu trữ] (https://app.supabase.io). Bạn có thể đăng ký và bắt đầu sử dụng Supabase mà không cần cài đặt bất kỳ thứ gì. Chúng tôi vẫn đang tạo ra trải nghiệm phát triển "local" - đây hiện là trọng tâm cốt lõi của chúng tôi, cùng với sự ổn định của nền tảng.
```
Can you help improve Supabase intro here @phucvo0708

The rest looks ok to me :+1:. Thank you

phamhieu replied:
> Supabase là một [nền tảng được lưu trữ]

sounds weird to me

phucvo0708 replied:
Sure, I'll take a quick look at that part.

phucvo0708 replied:
New changes were pushed to address the weird sounded translations that I made. Please take a look at it @phamhieu.

vhpx committed:
Update new translated strings

phamhieu commented on i18n/README.vi-vn.md:
```diff
@@ -0,0 +1,88 @@
+
+**Kiến trúc hiện tại**
+
+Supabase là một [nền tảng lưu trữ cơ sở dữ liệu] (https://app.supabase.io). Bạn có thể đăng ký và bắt đầu sử dụng Supabase mà không cần cài đặt bất kỳ thứ gì. Chúng tôi vẫn đang tạo ra trải nghiệm phát triển "local" - đây hiện là trọng tâm cốt lõi của chúng tôi, cùng với sự ổn định của nền tảng.
```
> We are still creating the local development experience

Chung toi van dang phat trien tinh nang chay Supabase trong moi truong ao hoa tren may tinh ca nhan

@phucvo0708 what do you think?

phamhieu replied:
it's still ok if we can't improve this. So difficult to translate

phucvo0708 replied:
> Chung toi van dang phat trien tinh nang chay Supabase trong moi truong ao hoa tren may tinh ca nhan

Sounds great to me, I'd like to make some modifications to that too.

"_Chúng tôi vẫn đang phát triển tính năng chạy Supabase trong môi trường ảo hoá trên_ **chính** _máy tính cá nhân_ **của bạn** _- đây hiện là trọng tâm cốt lõi của chúng tôi, cùng với sự ổn định của nền tảng._"

Just a small addition to your version, does that sounds better? @phamhieu

phamhieu commented:
lgtm! :rocket::tada:

vhpx committed:
Minor translation change

phucvo0708 commented:
Hey @phamhieu, the PR is looking good. We can make additional changes to the translations in the future if needed.

For now, let's merge this PR.
